src/OutsideClickHandler.jsx

```diff
@@ -112,7 +113,7 @@ export default class OutsideClickHandler extends React.Component {
     return (
       <div
         ref={this.setChildNodeRef}
-        style={display === DISPLAY.INLINE_BLOCK ? { display } : undefined}
+        style={Object.values(DISPLAY).includes(display) ? { display } : undefined}
```

Comment:
this is needlessly O(2n), this should be `has(DISPLAY, display)` instead; and part of the goal is that when it's "block", there's no style, since that's the default for divs

Reply:
Fair enough, didn't want to bring in lodash unnecessarily but if it's alright with you then sounds good to me.

Reply:
Absolutely not lodash; I meant the "has" package.

Reply:
not sure how `has` would work here as it checks with `hasOwnProperty`,  not for values, and the `display` prop being passed in would either be `block`, `inline-block`, or `flex` which are values not keys of the lookup table `DISPLAY`

Reply:
oh true, that's a good point :-)

ok in that case, i guess this is fine (we'd need to use https://npmjs.com/object.values tho, and not the builtin) - altho it's probably better to build up a reverse object at module level, and then use `has` here.